tests/test_make.py

```diff
@@ -337,6 +337,92 @@ class E(C, D):
             "d2='d2', e1='e1', e2='e2')"
         ) == repr(E())
 
+    def test_overwrite_proper_mro(self):
+        """
+        The proper MRO path works single overwrites too.
+        """
+
+        @attr.s(collect_by_mro=True)
+        class C(object):
+            x = attr.ib(default=1)
+
+        @attr.s(collect_by_mro=True)
+        class D(C):
+            x = attr.ib(default=2)
+
+        assert "D(x=2)" == repr(D())
+
+    def test_multiple_inheritance_proper_mro(self):
+        """
+        Attributes are collected according to the MRO.
+
+        See #428
+        """
+
+        @attr.s
+        class A(object):
+            a1 = attr.ib(default="a1")
+            a2 = attr.ib(default="a2")
+
+        @attr.s
+        class B(A):
+            b1 = attr.ib(default="b1")
+            b2 = attr.ib(default="b2")
+
+        @attr.s
+        class C(B, A):
+            c1 = attr.ib(default="c1")
+            c2 = attr.ib(default="c2")
+
+        @attr.s
+        class D(A):
+            d1 = attr.ib(default="d1")
+            d2 = attr.ib(default="d2")
+
+        @attr.s(collect_by_mro=True)
+        class E(C, D):
+            e1 = attr.ib(default="e1")
+            e2 = attr.ib(default="e2")
+
+        assert (
+            "E(a1='a1', a2='a2', d1='d1', d2='d2', b1='b1', b2='b2', c1='c1', "
+            "c2='c2', e1='e1', e2='e2')"
+        ) == repr(E())
+
+    def test_mro(self):
+        """
+        Attributes and methods are looked up the same way.
+
+        See #428
```

Comment:
```suggestion
        See https://github.com/python-attrs/attrs/issues/428
```
teensy tiny convenient lil' change

Reply:
Generally I would agree, however I refer to it multiple times and that would get really noisy. :|